packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -0,0 +1,631 @@
+      this.setColor('#000');
+    }
+  }
+
+  handleCopy() {
+    navigator.clipboard.writeText(this._input.value as string).then(() => {
+      // Show copied animation
+      this._previewButton.classList.add('color-picker__preview-color--copied');
+      this._previewButton.addEventListener('animationend', () => {
+        this._previewButton.classList.remove(
+          'color-picker__preview-color--copied'
+        );
+      });
+    });
+  }
+
+  openColorPicker(event: Event) {
+    event.stopImmediatePropagation();
+
+    const target = event.target as HTMLElement;
+    const popover = target.nextElementSibling as UUIPopoverElement;
+
+    popover.open = !popover?.open;
+
+    target.setAttribute('aria-expanded', popover.open.toString());
+  }
+
+  closeColorPicker(event: Event) {
+    const target = event.target as UUIPopoverElement;
+    const trigger = target.previousElementSibling;
+
+    if (trigger) {
+      trigger.setAttribute('aria-expanded', 'false');
+    }
+  }
+
+  handleEyeDropper() {
+    if (!hasEyeDropper) {
+      return;
+    }
+
+    const eyeDropper = new EyeDropper();
+
+    eyeDropper
+      .open()
+      .then(result => this.setColor(result.sRGBHex))
+      .catch(() => {
+        // The user canceled, do nothing
+      });
+  }
+
+  setColor(colorString: string | HslaColor) {
+    this._colord = new Colord(colorString);
+
+    const { h, l, s, a } = this._colord.toHsl();
+
+    this.hue = h;
+    this.saturation = s;
+    this.lightness = l;
+    this.alpha = this.opacity ? a * 100 : 100;
+
+    this._syncValues();
+
+    this.dispatchEvent(
+      new UUIColorPickerChangeEvent(UUIColorPickerChangeEvent.CHANGE)
+    );
+    return true;
+  }
+
+  setLetterCase(string: string) {
+    if (typeof string !== 'string') {
+      return '';
+    }
+    return this.uppercase ? string.toUpperCase() : string.toLowerCase();
+  }
+
+  private _syncValues() {
+    this.inputValue = this.getFormattedValue(this.format);
+    this.value = this.inputValue;
+  }
+
+  private _renderColorPicker() {
+    return html`
+      <div
+        class=${classMap({
+          'color-picker': true,
+          'color-picker--inline': this.inline,
+          'color-picker--disabled': this.disabled,
+        })}
+        aria-disabled=${this.disabled ? 'true' : 'false'}>
+        <uui-color-area .value="${this.value}" @change=${this.handleGridChange}>
+        </uui-color-area>
+        <div class="color-picker__controls">
+          <div class="color-picker__sliders">
+            <uui-color-slider
+              label="hue"
+              class="hue-slider"
+              .value=${Math.round(this.hue)}
+              @change=${this.handleHueChange}>
+            </uui-color-slider>
+            ${this.opacity
+              ? html`
+                  <uui-color-slider
+                    label="alpha"
+                    class="opacity-slider"
+                    .value=${Math.round(this.alpha)}
+                    type="opacity"
```

Comment:
It does exist :)  
That is one of the things I added to color slider - there are two predefined types on it - a hue slider and opacity slider, that gives you backgrounds and predefined min/max values. Two more could be implemented - `saturation` and `lightness`. There is even more potencial here for `red`, `green`, `blue` sliders and so on.  I didnty bother with that now as none of it is part of the color picker, but with that we could have a fully funcional color slider wich allows to easily edit different color values. I can open an issue with that so it can be up-for-grabs

Reply:
Hmm apologies if I didn't find it. My IDE complained that it was an unknown property. Your link seems to lead to the "disabled" attribute?

Reply:
Sorry, I fixed the link. I might need another coffee

Reply:
Ahh great. I think we just need to have it decorated with jsdoc so it can be found. I'm not sure. Could you check if it's documented all right in the custom-components.json file all right if you build the package?